lib/poetry-lock/pyproject.toml

```diff
@@ -28,15 +28,15 @@ networkx = "3.2.1"
 ipython = "8.18.1"
 
 # Package versions from being yanked
-zope-interface = ">=6.4.post0"
+zope-interface = "7.1.0"
```

Comment:
I'd do >=7.1.0 here so we don't force people to have zope-interface 7.1.0
Also this should be in a different section now, I'm assuming just below 7.1.0 wasn't yanked.

It's possible you can just remove it.  If we need 7.1.0 for recency, the dependency should be in some other repo.

Reply:
Removed.